src/playground/public/assets/js/demos.ts

```diff
@@ -0,0 +1,25 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+function generateDemoMenuEntries(editor: AceAjax.Editor) {
+  const menu = document.getElementById('demos-menu')!;
+
+  for (const key in demos) {
+    const a = document.createElement('a');
+    a.setAttribute('class', 'dropdown-item');
+    a.setAttribute('href', '#');
+    a.onclick = (e) => {
+      e.preventDefault();
+      pickDemo(editor, key);
+    };
+    a.innerText = demos[key].title;
+    menu.appendChild(a);
+  }
```

Comment:
Not sure why you're not using Handlebars for this https://handlebarsjs.com/guide/builtin-helpers.html#each... Also, templates for frontend are quite old-fashioned